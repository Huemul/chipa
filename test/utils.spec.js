const { readFile } = require('../utils')

describe('readFile', () => {
  it('resolves with the content of a file', () => {
    return readFile('./test/no-code.md')
      .then(content => {
        expect(content).toMatchSnapshot()
      })
  })

  it('rejects with the ENOET error', () => {
    return readFile('./test/non-existent.txt')
      .catch(error => {
        expect(error).toMatchSnapshot()
      })
  })
})

