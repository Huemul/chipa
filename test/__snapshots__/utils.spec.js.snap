// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`readFile rejects with the ENOET error 1`] = `[Error: ENOENT: no such file or directory, open './test/non-existent.txt']`;

exports[`readFile resolves with the content of a file 1`] = `
"# an md file with no code

> no snippets should be extracted from this file
"
`;
